packages/onchain-claims/package.json

```diff
@@ -58,6 +57,7 @@
     "@openzeppelin/contracts": "4.8.0",
     "@openzeppelin/contracts-upgradeable": "4.8.0",
     "@openzeppelin/truffle-upgrades": "1.17.0",
+    "@poanet/solidity-flattener": "^3.0.9",
```

Comment:
this is misleading because the version is bumped here, but this dependency is not upgraded (no change to the lockfile). So, it's better to use `^3.0.7` here or keep `^3.0.9` and upgrade this package

Reply:
AFAIK only `package-lock` at root level needs to be maintained. There are mixed opinions on this topic, I will update as you said.

Reply:
but, there is also a `package-lock.json` in this repository, so if we move to node v18 here, version 3.0.7 enforced by the lockfile will complain to install when developing locally or in CI

Reply:
These `package-lock` files are ignored and only root level is considered is what I have read.